README.md

```diff
@@ -216,6 +207,30 @@ integration-shell:me@somewhere:haskell-backend$ exit
 me@somewhere:haskell-backend$ 
 ```
 
+### Integration/Performance tests of downstream projects
+
+#### scripts/performance-tests-{kevm, kontrol}.sh
+
+Call these scripts from the root of the repo to obtain performance numbers for the KEVM and Kontrol test suites. These are necessary for any new feature which is expected to modify the perfromance of the booster and the timings should be includedf in the PR.
+
+
+#### scripts/booster-analysis.sh
+
+This scipt can be used with any folder containing bug reports to build an anlysis of fallback/abort reasons in the booster. To obtain bug reports, first run `PYTEST_PARALLEL=8 scripts/performance-tests-kevm.sh --bug-report`, which will generate tarballs for all the tests and drop them into `scripts/bug-reports/`. Then call `scripts/booster-analysis.sh scripts/booster-analysis.sh scripts/bug-reports/kevm-v1.0.417-main`
+
+
+### Running integration tests with Docker
+
+Use `docker.sh` to run commands inside the container:
+
+``` sh
+./docker/build.sh  # run once when dependencies change
+./docker/run.sh make all  # build the backend
+./docker/run.sh make test  # run all tests
+./docker/run.sh make -C test/imp test  # run all tests in test/imp
+```
+
+
```

Comment:
The docker setup is rather outdated, and was not optimal in the first place IMHO (installing `stack` fresh into a K framework image is the wrong way round given the rate of change). We could of course look at how the CI uses docker files as we spent some time making those. However, I am in favour of just deleting `Dockerfile` and `docker/` and not supporting dockerised testing any more.
```suggestion
```